Revision Tests/maxi_stepanov.md

```diff
@@ -0,0 +1,32 @@
+## Please translate into English
+
+**_1. Если бы я был умнее (по жизни) то я бы сделал вчера бэкап всей базы и только потом применял свой ПР к коду на проде. Жаль что я не умный как мой папа._**
+
+> If I’d had smart, I would have done the backup yesterday and the apply my PR to the production.  I wish I was smart like my father.
```

Comment:
If I were smarter then I would have made the whole DB backup yesterday and only then merged my Pr in prod. I wish I were smart as my father.